exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/efgs/CallbackController.java

```diff
@@ -0,0 +1,34 @@
+package fi.thl.covid19.exposurenotification.efgs;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import static net.logstash.logback.argument.StructuredArguments.keyValue;
+
+@RestController
+@RequestMapping("/efgs")
+public class CallbackController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CallbackController.class);
+
+    private final FederationGatewayService federationGatewayService;
+
+    public CallbackController(FederationGatewayService federationGatewayService) {
+        this.federationGatewayService = federationGatewayService;
+    }
+
+    @GetMapping("/callback")
+    public void triggerCallback(@RequestParam("batchTag") String batchTag, @RequestParam("date") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate date) {
```

Comment:
I find it odd that they decided to go with no-zone local dates on an international API... but I guess it is what it is. Is it a documented thing at least that it's interpreted as UTC?

Reply:
From code:

`.queryParam("date", callbackTask.getBatch().getCreatedAt()
        .withZoneSameInstant(ZoneOffset.UTC).format(DateTimeFormatter.ISO_LOCAL_DATE))`